misk-rate-limiting-bucket4j-dynamodb-v1/src/main/kotlin/misk/ratelimiting/bucket4j/dynamodb/v1/DynamoDbV1Bucket4jRateLimiterModule.kt

```diff
@@ -16,8 +17,9 @@ import java.time.Clock
 /**
  * Configures a [RateLimiter] that uses DynamoDb as a backend.
  */
-class DynamoDbV1Bucket4jRateLimiterModule(
-  private val tableName: String
+class DynamoDbV1Bucket4jRateLimiterModule @JvmOverloads constructor(
+  private val tableName: String,
+  private val prunerPageSize: Int = 1000
```

Comment:
Could you add some docs around the tradeoff of making `prunerPageSize` bigger or smaller?

Reply:
Can do 👍